src/results-main.js

```diff
@@ -116,8 +116,13 @@
   var createCharts = function(resp) {
     if (isCanvasSupported()) {
       $(".display-options").addClass("display-inline");
-      var contests = resp.AllElections[0].contests;
-      charts = arrayMap(contests, makeElectionChart);
+	  const {
```

Comment:
so unfortunately we don't have "es6" support in this project, so i'm pretty sure the const and object destructing will fail in ie11 (yes we still support that).

A way around this would be to create a function that you could do these checks.


```js

/** Ensure we have 
var getContestsFromResponse = function(response) {
  ...
}
```

Where the return value is an array of contest, even if it's an empty array.

Does that make sense?

Reply:
@srutherfordbcg you could also argue with me to just add support for transpiling, it looks like parcel might do this rather easily https://parceljs.org/transforms.html, let me know if you would prefer to do.

Reply:
disregard all this nonsense, this works in ie11 because parcel transpiles it automatically.

Reply:
@srutherfordbcg it does look like you repeat

```js
const {
		AllElections = []
	} = resp;
	const {
		contests = []
	} = AllElections.length > 0 ? AllElections[0] : [];
```

You could conver this to a function

so maybe something like

```js
const getContentsFromResponse = response => {
  const { 
            AllElections = []
	  } = resp;
	 return {
		  contests = []
	  } = AllElections.length > 0 ? AllElections[0] : [];
};
```

This may limit your usage, because you also need access to AllElections, but I think you get the idea.